src/systemtest/resources/features/ArtifactFlow.feature

```diff
@@ -0,0 +1,15 @@
+@AuthFeature
+Feature: Artifact system test
+
+  @ArtifactFlowScenario
+  Scenario: Artifact Flow Scenario
+    Given configurations are provided
+    And some subscriptions are set up in another story etc
```

Comment:
Even if it's temporary I don't like the woding of this AND. Remove the last part, and keep only "some subscriptions are set up"

Reply:
we do not need to check that the subscriptions are set up. Remove this line completely.

Reply:
This is not to check. We still need some place to add the subscriptions to EI i guess?